gradle/owasp-suppression.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
     <!-- See https://jeremylong.github.io/DependencyCheck/general/suppression.html for examples -->
-    <suppress until="2023-11-01">
+    <suppress until="2023-12-12">
         <notes><![CDATA[
         Temporary suppression, as it's arguably a false positive: https://github.com/netty/netty/issues/8537#issuecomment-1527896917
         The correct thing to do is ensure hostname verification is enabled: https://codetinkering.com/enable-hostname-verification-netty/
         This is indeed the case for Azure, which is the netty-handler: https://github.com/Azure/azure-sdk-for-java/issues/18286#issuecomment-947958978
+        Also see: https://github.com/jeremylong/DependencyCheck/issues/5912
         The other vulnerable lib is besu-metrics, which might want looking at, will see if this CVE gets flagged in besu first
         file name: netty-handler-4.1.97.Final.jar
         ]]></notes>
-        <packageUrl regex="true">^pkg:maven/io\.netty/netty\-handler@.*$</packageUrl>
```

Comment:
What was the reason for loosening the package filter? I'd be concerned that this CVE might crop up in other packages even though we've only considered netty-handler

Reply:
updated with more generic netty expression.